epsie/chain/chain.py

```diff
@@ -414,6 +414,15 @@ def clear(self):
         self._lastclear = self._iteration
         return self
 
+    def _reset_proposals(self):
```

Comment:
Don't make this a hidden function (remove the leading `_`). A user may find it useful to reset at will; no reason to hide that functionality.